.github/automation/commit-msg-check.py

```diff
@@ -79,7 +67,7 @@ def main():
       is_ok = is_ok and result
 
     if is_ok:
```

Comment:
Casually switching to snake_case halfway through the file that started with camelCase..

Reply:
Looks like argparse package set the tone for `main()`...